src/netcdf_io/calc_analysis.fd/inc2anl.f90

```diff
@@ -277,12 +277,14 @@ subroutine add_increment(fcstvar, incvar)
           incncfile = open_dataset(incr_file, paropen=.true.)
           ! JEDI-derived increments have a time dimension, so read to appropriate array
           if ( jedi ) then
-             call read_vardata(incncfile, trim(incvar)//"_inc", work3d_inc_jedi, nslice=k, slicedim=3)
-             ! add increment to background
-             do j=1,nlat
-                jj=nlat+1-j ! increment is S->N, history files are N->S
-                work3d_bg(:,j,1) = work3d_bg(:,j,1) + work3d_inc_jedi(:,jj,1)
-             end do
+             if (has_var(incncfile, trim(incvar)//"_inc")) then
+                call read_vardata(incncfile, trim(incvar)//"_inc", work3d_inc_jedi, nslice=k, slicedim=3)
+                ! add increment to background, otherwise do nothing
+                do j=1,nlat
+                   jj=nlat+1-j ! increment is S->N, history files are N->S
+                   work3d_bg(:,j,1) = work3d_bg(:,j,1) + work3d_inc_jedi(:,jj,1)
+                end do
+             endif
           else
              call read_vardata(incncfile, trim(incvar)//"_inc", work3d_inc_gsi, nslice=k, slicedim=3)
```

Comment:
Will this need to be wrapped in a similar `has_var` if block to prevent reading the `omga` variable for non-JEDI runs?

Reply:
Based on @XiaqiongZhou-NOAA ' s reply, `omga` has been removed from `calc_analysis` output.  Done at 4eda78d.